source/renderer/app/components/staking/epochs/StakingEpochs.scss

```diff
@@ -1,5 +1,174 @@
+%progressLabelText {
+  font-family: var(--font-semibold);
+  font-size: 14px;
+  line-height: 24px;
+  position: absolute;
+  top: 0;
+  white-space: nowrap;
+}
+
 .component {
-  font-family: var(--font-regular);
-  margin: 20px;
-  text-align: center;
+  padding: 20px 20px 0;
+
+  .headerWrapper {
+    align-items: center;
+    display: flex;
+    justify-content: center;
+    margin: 0 20px 6px;
+  }
+
+  .epochSelector {
+    width: 275px;
+
+    input {
+      border: 0;
+      font-family: var(--font-semibold);
+      font-size: 19px;
+      line-height: 1.37;
+      padding-left: 0;
+      padding-right: 0;
+      padding-top: 0;
+    }
+  }
+
+  .noResultsLabel {
+    color: var(--theme-staking-font-color-lighter);
+    flex-grow: 1;
+    font-family: var(--font-regular);
+    font-size: 16px;
+    text-align: center;
+  }
+
+  .loadingSpinnerWrapper {
+    margin: auto;
+  }
+
+  .currentEpochProgressBar {
+    height: 24px;
+    margin-bottom: 31px;
+    margin-left: 10px;
+    margin-right: 10px;
+    margin-top: -2px;
+  }
+
+  .progressBarContainer {
+    background-color: var(--theme-staking-background-color);
+    border-radius: 5px;
+    height: 100%;
+    position: relative;
+    width: 100%;
+  }
+
+  .progress {
+    background: repeating-linear-gradient(
+      -63deg,
+      var(--theme-staking-progress-stripe-dark-1-background-color),
+      var(--theme-staking-progress-stripe-dark-1-background-color) 10px,
+      var(--theme-staking-progress-stripe-dark-2-background-color) 10px,
+      var(--theme-staking-progress-stripe-dark-2-background-color) 20px
+    );
+    border-radius: 5px;
+    height: 100%;
+    overflow: hidden;
+    position: relative;
+  }
+
+  .overlapProgressLabel {
+    @extend %progressLabelText;
+    color: #fff;
+    transform: translateX(calc(-100% - 8px));
+    z-index: 1000;
+  }
+
+  .progressLabel {
+    @extend %progressLabelText;
+    color: var(--theme-staking-font-color-regular);
+    right: 8px;
+    z-index: 500;
+  }
+
+  table {
+    border-style: hidden;
+    user-select: text;
```

Comment:
@yakovkaravelov are we sure we don't want the contents of the table to be selectable?
What do you think @daniloprates @DmitriiGaico?

Reply:
@daniloprates @nikolaglumac The table shows % so it's not a precise information and i don't think someone will need to Copy&Past such data. So my point of view to keep it unselectable.

Reply:
OK. That makes sense. Thanks @DmitriiGaico!